src/web/rest_api/common/time.py

```diff
@@ -7,3 +7,20 @@
 
 def utc_now_microseconds():
     return int(datetime.datetime.utcnow().timestamp() * 1e6)
+
+def iso_date_microseconds(iso_date: str):
```

Comment:
Do we still need this now that we're using the UNIX microseconds? If not I'd suggest removing (I want to keep the lines of code count in the REST API as low as possible)